Ports/optipng/package.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="optipng"
+version="0.7.7"
+files="http://prdownloads.sourceforge.net/optipng/optipng-${version}.tar.gz?download optipng-${version}.tar.gz 4f32f233cef870b3f95d3ad6428bfe4224ef34908f1b42b0badf858216654452"
```

Comment:
That link seems to be skipping a load balancer (or at least it's not the link that the downloads overview redirects to). `https://downloads.sourceforge.net/optipng/optipng-${version}.tar.gz` seems to work just fine as well.

Reply:
Strange, that's the exact link the website provides